### PR TITLE
Added Circle assigment

### PR DESCRIPTION
**Pull request by nallarun10**

### CODE REVIEW

File: students/arun_nalla/session08/circle.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+""" Circle class assignment by
+Arun Nalla 12/08/2018"""
+
+from math import pi
+import random
+
+class Circle():
+
+    def __init__(self, radius):
+        if radius >= 1:
```
Comment (hosungs) [inline students/arun_nalla/session08/circle.py]: Shouldn't this be `if radius > 0:`, if you don't want to allow 0-radius circle? Note that radius doesn't have to be integer. It's a minor point, though, I suppose...

File: students/arun_nalla/session08/circle.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            self.radius = radius
+        else:
+            raise ValueError ('Value is less than or equal to zero')
+
+    @property
+    def diameter(self):
+        self.diameter = self.radius * 2
+        return self.radius
```
Comment (hosungs) [inline students/arun_nalla/session08/circle.py]: Why return `self.radius`, after computing `self.diameter`? Also you don't need to save the computed result to an attribute (`self.diameter`). You can just return the computed value as is (`return self.radius * 2`).

**Review by hosungs — COMMENTED** [review]
Left a few comments (minor ones that better be addressed). Will wait for them to be addressed, before merging/closing this PR. Thanks.

File: students/arun_nalla/session08/circle.py (hunk 1 above)
Comment (nallarun10) [reply students/arun_nalla/session08/circle.py]: Thanks Housung, added revised file.

**Review by hosungs — APPROVED** [review]
Thanks for your revision. Merging/closing this PR as such.

**Pull request merged**